frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/nodes-selection-list.tsx

```diff
@@ -156,18 +119,9 @@ export const NodesSelectionList: React.FC<NodesSelectionListProps> = (props) =>
 type NodesSelectionListProps = {
   data: NodeKind[];
   customData: {
-    allSelected: boolean;
-    rows: RowUIDMap;
-    setAllSelected: React.Dispatch<React.SetStateAction<boolean>>;
-    setRows: React.Dispatch<React.SetStateAction<RowUIDMap>>;
+    selectedNodes: NodeKind[];
+    setSelectedNodes: React.Dispatch<React.SetStateAction<NodeKind[]>>;
+    visibleRows: NodeKind[];
+    setVisibleRows: React.Dispatch<React.SetStateAction<NodeKind[]>>;
```

Comment:
This should be common across all the Components that use this hook. A common type declaration would be better.
`customData: CustomData<NodeKind>`
```
type CustomData <R extends K8sResourceCommon = K8sResourceKind> = {
 selectedRows: R[];
...
```